### PR TITLE
LXD fixes

### PR DESCRIPTION
**Pull request by dann1**
- Improved/fixed logging
- Fixed storage issues
- Fixed reverted BASE_PATH detection

### CODE REVIEW

File: src/vmm_mad/remotes/lib/lxd/mapper/mapper.rb [hunk 1]
```diff
@@ -185,40 +185,42 @@ def unmap(one_vm, disk, directory)
         sys_parts = lsblk('')
 
         return false unless sys_parts
- 
+
         partitions = []
         device = ''
-        real_path = directory
+
+        dir_ok = directory.dup.gsub('//', '/')
```
Comment (rsmontero) [inline src/vmm_mad/remotes/lib/lxd/mapper/mapper.rb]: Why is this needed? Double // should be ok in a path. Does it break any addtional parsing?

Comment (dann1) [reply src/vmm_mad/remotes/lib/lxd/mapper/mapper.rb]: > Why is this needed? Double // should be ok in a path. Does it break any addtional parsing?

because of [this](https://github.com/dann1/one/blob/e55c247c24cff72b0bd054e3be1d2be01457c8ed/src/vmm_mad/remotes/lib/lxd/mapper/mapper.rb#L202)
